base/setup/usetup/lang/he-IL.h

```diff
@@ -1478,6 +1478,11 @@ MUI_ERROR heILErrorEntries[] =
         "  \x07  .\204\220\227\232\204\204\216 \232\200\226\214 \211\203\213 F3 \225\207\214",
         "\204\220\227\232\204 \214\205\210\211\201 = F3  \212\231\216\204 = ENTER"
     },
+    {
+        // ERROR_NO_BUILD_PATH
+        "Failed to build the installation paths for the ReactOS installation directory!\n"
+        "ENTER = Reboot computer"
```

Comment:
If you wouldn't mind, can you change to this:
`!ReactOS \214\231 \204\220\227\232\204\204 \232\211\227\211\232 \230\205\201\222 \204\211\227\211\232 \211\201\211\232\220 \232\211\211\220\201\201 \204\214\231\213\220 \204\220\227\232\204\204 \232\211\220\213\232\n`
`\201\231\207\216\204 \232\200 \231\203\207\216 \214\222\224\204 = ENTER`
Thanks.

Reply:
Sorry, that goes outside the purpose of what is this PR meant for. Could you please at least provide a screenshot confirming that the encoding is done properly?

Reply:
How I can trigger this error?